decisions/wc_society_decisions.txt

```diff
@@ -438,4 +478,444 @@ targeted_decisions = {
 #			factor = 0
 #		}
 #	}
-#}
\ No newline at end of file
+}
+society_decisions = {
+	ask_for_ancient_mark = {
+
+		ai_check_interval = 36
+
+		only_playable = yes
+
+		potential = {
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+			society_member_of = cenarion_circle
+		}
+
+		allow = {
+			is_inaccessible_trigger = no
+			society_currency >= 500
+			custom_tooltip = {
+				text = is_not_busy_trigger_tooltip
+				hidden_tooltip = { NOT = { has_character_flag = do_not_disturb } }
+			}
+			custom_tooltip = {
+				text = has_not_used_decision_3_years_tooltip
+				hidden_tooltip = {
+					OR = {
+						NOT = { has_character_flag = ask_for_ancient_mark_cooldown_flag }
+						had_character_flag = { flag = ask_for_ancient_mark_cooldown_flag years = 3 }
+					}
+				}
+			}
+		}
+
+		effect = {
+			set_character_flag = do_not_disturb			# Always must be before event chain
+			
+			character_event = { id = WCCSO.10100 }
+			change_society_currency = -500
+		}
+
+		revoke_allowed = {
+			always = no
+		}
+
+		ai_will_do = {
+			factor = 1
+		}
+	}
+	train_class_society = {
+
+		ai_check_interval = 120
+
+		only_playable = yes
+
+		potential = {
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+			OR = {
+				society_member_of = cenarion_circle
+				society_member_of = sisterhood_of_elune
+			}
+			OR = {
+				society_rank > 1
+				society_member_of = sisterhood_of_elune
```

Comment:
Instead of 2 OR, how about this?
```
OR = {
	AND = {
		society_member_of = cenarion_circle
		society_rank > 1
	}
	society_member_of = sisterhood_of_elune
}

Reply:
implemented